Userland/Applications/ImageViewer/ViewWidget.h

```diff
@@ -36,6 +36,8 @@ class ViewWidget final : public GUI::AbstractZoomPanWidget {
     bool scaled_for_first_image() { return m_scaled_for_first_image; }
     void set_scaled_for_first_image(bool val) { m_scaled_for_first_image = val; }
     void set_path(DeprecatedString const& path);
+    void set_forced_dimensions(Gfx::IntSize dimensions);
```

Comment:
You seem to be using this API for two purposes:
- read in binary data for specific dimensions;
- crop a loaded bitmap to specific dimensions.

What if I want to do both? I.e. load a binary bitmap with specific dimensions, then crop them? I think this should be two separate APIs and two separate command line arguments. And in this case, the newly added functionality of cropping an image in ImageViewer (for whatever reason we may want to do so) should be a separate commit.

Ideally this also gets rid of all the `.clear()`s that are now required.